### PR TITLE
Scan: patch for fast-sync db and guanyu migration db

### PR DESCRIPTION
**Pull request by evilpeach**
## Implementation details
- patch subscription for fast sync DB and new GuanYu DB

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: scan/src/components/request/DataReports.re [hunk 1]
```diff
@@ -34,11 +34,7 @@ let make = (~reports: array(RequestSub.report_t)) => {
       ->Belt.Array.mapWithIndex(
           (
             idx,
-            {
-              reportValidator: {operatorAddress, moniker, identity},
-              transaction: {hash},
-              reportDetails,
-            },
+            {reportValidator: {operatorAddress, moniker, identity}, transaction, reportDetails},
```
Comment (pzshine) [inline scan/src/components/request/DataReports.re]: Can you change `transaction` to `optTransaction`?

File: scan/src/subscriptions/RequestSub.re [hunk 2]
```diff
@@ -126,7 +126,7 @@ module Mini = {
           clientID: client_id
           requestTime: request_time @bsDecoder(fn: "GraphQLParser.fromUnixSecondOpt")
           resolveTime: resolve_time @bsDecoder(fn: "GraphQLParser.fromUnixSecondOpt")
-          sender @bsDecoder(fn: "Address.fromBech32")
+          sender
```
Comment (pzshine) [inline scan/src/subscriptions/RequestSub.re]: Change it into `addressExn` from `GraphQLParser`

File: scan/src/subscriptions/RequestSub.re [hunk 3]
```diff
@@ -79,7 +79,7 @@ module Mini = {
                clientID: client_id
                requestTime: request_time @bsDecoder(fn: "GraphQLParser.fromUnixSecondOpt")
                resolveTime: resolve_time @bsDecoder(fn: "GraphQLParser.fromUnixSecondOpt")
-               sender @bsDecoder(fn: "Address.fromBech32")
+               sender
```
Comment (pzshine) [inline scan/src/subscriptions/RequestSub.re]: Change it into `addressExn` from `GraphQLParser`

File: scan/src/subscriptions/RequestSub.re [hunk 4]
```diff
@@ -167,7 +167,7 @@ module Mini = {
           clientID: client_id
           requestTime: request_time @bsDecoder(fn: "GraphQLParser.fromUnixSecondOpt")
           resolveTime: resolve_time @bsDecoder(fn: "GraphQLParser.fromUnixSecondOpt")
-          sender @bsDecoder(fn: "Address.fromBech32")
+          sender
```
Comment (pzshine) [inline scan/src/subscriptions/RequestSub.re]: Change it into `addressExn` from `GraphQLParser`

File: scan/src/subscriptions/RequestSub.re [hunk 5]
```diff
@@ -256,16 +257,22 @@ module Mini = {
          ->Belt_Array.map(y =>
              {
                id: y##id,
-               sender: y##sender,
+               sender: y##sender->Belt.Option.getExn->Address.fromBech32,
```
Comment (pzshine) [inline scan/src/subscriptions/RequestSub.re]: Change it into `addressExn` from `GraphQLParser`

File: scan/src/subscriptions/RequestSub.re [hunk 6]
```diff
@@ -317,16 +324,22 @@ module Mini = {
...
```
Comment (pzshine) [inline scan/src/subscriptions/RequestSub.re]: Change it into `addressExn` from `GraphQLParser`

File: scan/src/subscriptions/RequestSub.re [hunk 7]
```diff
@@ -461,7 +526,7 @@ module SingleRequestConfig = [%graphql
         }
         minCount: min_count
         resolveStatus: resolve_status  @bsDecoder(fn: "parseResolveStatus")
-        requester: sender @bsDecoder(fn: "Address.fromBech32")
+        sender
```
Comment (pzshine) [inline scan/src/subscriptions/RequestSub.re]: Change it into `addressExn` from `GraphQLParser`

File: scan/src/subscriptions/RequestSub.re [hunk 8]
```diff
@@ -529,7 +594,7 @@ module MultiRequestConfig = [%graphql
...
```
Comment (pzshine) [inline scan/src/subscriptions/RequestSub.re]: Change it into `addressExn` from `GraphQLParser`

**Review by pzshine — CHANGES_REQUESTED** [review]

**Review by pzshine — APPROVED** [review]
Good to go!

**Review by Pitchanai — APPROVED** [review]

**Pull request merged**